internal/commands/set_run_image_mirrors_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 func TestSetRunImageMirrorsCommand(t *testing.T) {
-	spec.Run(t, "Commands", testSetRunImageMirrorsCommand, spec.Parallel(), spec.Report(report.Terminal{}))
+	spec.Run(t, "Commands", testSetRunImageMirrorsCommand, spec.Sequential(), spec.Report(report.Terminal{}))
```

Comment:
I'm sort of confused by this as it seems we use `spec.Parallel()` more often than `spec.Sequential()`. What's the reason that this test needs to be run sequentially?

Reply:
This test is uses environment variables during setup [here](https://github.com/buildpacks/pack/blob/5b6731320073fe1451f440ac93c27edf2f792a2a/internal/commands/set_run_image_mirrors_test.go#L41) and [here](https://github.com/buildpacks/pack/blob/5b6731320073fe1451f440ac93c27edf2f792a2a/internal/commands/set_run_image_mirrors_test.go#L45), environment variables are shared by goroutines so running this test with any sort of parallelism will have race conditions. Using `make unit` runs unit tests sequentially by using a single goroutine configured [here](https://github.com/dwillist/pack/blob/5b6731320073fe1451f440ac93c27edf2f792a2a/Makefile#L5) so they end up passing.